manifests/vhost.pp

```diff
@@ -16,12 +16,15 @@
   Optional[String]             $content = undef,
 ) {
 
+  include caddy
+
   file { "/etc/caddy/config/${title}.conf":
     ensure  => file,
     content => $content,
     source  => $source,
+    owner   => $caddy::caddy_user,
+    group   => $caddy::caddy_group,
     mode    => '0444',
-    require => Class['caddy::config'],
```

Comment:
This config was required to ensure `/etc/caddy/config` exists, but that will happen anyway due to automatic relationships.

Reply:
I think this require should make sure that the config ```/etc/caddy/Caddyfile``` is available.

Reply:
Good catch, adding that.